### PR TITLE
multiple delete keys

### PR DESCRIPTION
**Pull request by saggiyogesh**
New pull request for multiple delete keys

### CODE REVIEW

**Comment by mrister** [comment]
@saggiyogesh Please write tests for multi key feature.

File: index.js [hunk 1]
```diff
@@ -267,6 +267,27 @@ function redisStore(args) {
...
+      key.forEach(function(k) {
+        args.push(k);
+      });
+
+    } else {
+      args.push(key);
+    }
+
+    args.push(handleResponse(conn, cb));
+    conn.del.apply(conn, args);
```
Comment (PuKoren) [inline index.js]: CircleCI raised some error here, as `conn` is not defined

Comment (mrister) [inline index.js]: yeah it needs ot be moved to connect function callback argument

**Comment by jeff-kilbride** [comment]
@mrister  I agree. Currently, I am using `Cache.store.getClient` and `redis.client.mset` / `redis.client.mget` when I want to set / get multiple keys. It would be much nicer to pass an array or an object to `Cache.set` / `Cache.get`, instead of using the underlying connection.

One issue that's kind of a pain is that there is no easy way to set a `ttl` on multiple keys. There is no `msetex` method, like there is with `setex`. So, when setting multiple keys with a `ttl`, you'd have to loop through each one using `expire` -- `expire` doesn't accept multiple keys either. Along with the compression and JSON parsing, it could get messy quickly. The `del` method is by far the easiest to implement of the three.

**Comment by mrister** [comment]
@jeff-kilbride Thanks for the research and details. Maybe we can open up an issue (feature) for this so  someone ca work on it once they have time and we can discuss the implementation approach on it?

**Comment by mrister** [comment]
It was implemetned in other pr

**Pull request closed without merging**